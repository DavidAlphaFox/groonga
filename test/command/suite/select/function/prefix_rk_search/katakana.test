table_create Kana TABLE_PAT_KEY ShortText --normalizer NormalizerAuto

table_create Items TABLE_HASH_KEY ShortText
column_create Items kana COLUMN_VECTOR Kana

column_create Kana items COLUMN_INDEX Items kana

load --table Items
[
{"_key": "Groonga", "kana": ["グルンガ"]}
]

select Items --filter 'prefix_rk_search(kana, "グル")'
