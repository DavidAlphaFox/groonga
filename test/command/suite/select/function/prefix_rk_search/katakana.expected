table_create Kana TABLE_PAT_KEY ShortText --normalizer NormalizerAuto
[[0,0.0,0.0],true]
table_create Items TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Items kana COLUMN_VECTOR Kana
[[0,0.0,0.0],true]
column_create Kana items COLUMN_INDEX Items kana
[[0,0.0,0.0],true]
load --table Items
[
{"_key": "Groonga", "kana": ["グルンガ"]}
]
[[0,0.0,0.0],1]
select Items --filter 'prefix_rk_search(kana, "グル")'
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "_key",
          "ShortText"
        ],
        [
          "kana",
          "Kana"
        ]
      ],
      [
        1,
        "Groonga",
        [
          "グルンガ"
        ]
      ]
    ]
  ]
]
